### PR TITLE
fix: fix placeholders on the new home view

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [] 

### Description

Resolves [Notion card](https://www.notion.so/artsy/Placeholders-of-some-of-the-components-are-incorrect-10ecab0764a080c0b726e3f584bb4599)
Fix placeholders on the new home view

| Activity Placeholder Before | Activity Placeholder After | 
| --- | --- |
| image | image |

| Artworks Rail Placeholder Before | Artworks Rail Placeholder After | 
| --- | --- |
| image) | image |

| Artists Rail Placeholder Before | Artists Rail Placeholder After |
| --- | --- |
| image | image |

| Articles Placeholder Before | Articles Placeholder After |
| --- | --- |
| <video src='https://github.com/user-attachments/assets/b659379d-e330-4604-a4a4-4964f4dfb0ee' width=180/>  | <video src='https://github.com/user-attachments/assets/29739084-bbb8-4253-a82e-5fdfa227cd28' width=180/> |

| Auction Results Placeholder Before | Auction Results Placeholder DimentionsAfter |
| --- | --- |
| image | image | 

| Fairs Placeholder Before | Fairs Placeholder After | 
| --- | --- |
| image | image |

| Shows for you Placeholder Before | Shows for you Placeholder After | 
| --- | --- |
| image | image |

Placeholders that do not require a change:
- HomeViewSectionGalleriesPlaceholder
- HomeViewSectionGeneric component
- HomeViewSectionHeroUnits
- HomeViewSectionMarketingCollections
- HomeViewSectionViewingRooms
- HomeViewSectionArticlesCardsPlaceholder
### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix placeholders on the new home view - daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix placeholders on the new home view - daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 609fcb85bcf47d935390d91306d2132f6c39d192
</p>

File: src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx [hunk 1]
```diff
@@ -173,18 +173,22 @@ const HomeViewSectionAuctionResultsPlaceholder: React.FC<FlexProps> = (flexProps
...
+                        <SkeletonBox width={150} height={15} />
+                        <SkeletonBox width={120} height={15} />
+                        <SkeletonBox width={200} height={15} />
+                      </Join>
+                    </Flex>
+
+                    <Flex>
+                      <Join separator={<Spacer y={0.5} />}>
+                        <SkeletonBox width={120} height={15} />
+                        <SkeletonBox width={80} height={22} />
```
Comment (anandaroop) [inline src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx:189]: **question:** Very minor, but should the second line here be the same height as the first? Seems like those two lines have the same xs font size (if I'm referencing the right UI here)

| Skeleton | UI |
|--------|--------|
 |  | |

**Review by anandaroop — APPROVED** [review]
Looking great 🙏🏽

File: src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx (hunk 1 above)
Comment (evaschilken) [reply src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx:189]: @MounirDhahri Could you take this one over since daria is out? 🙏🏻

Comment (MounirDhahri) [reply src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx:189]: sure

**Review by MounirDhahri — APPROVED** [review]

File: src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/HomeView/Sections/HomeViewSectionAuctionResults.tsx]: Thanks for taking it over @MounirDhahri! Missed the handover of this one, sorry!

**Pull request merged**